packages/functions/src/repositories/taskRepository.ts

```diff
@@ -19,7 +19,8 @@ export default class TaskRepository{
         const params = {
             TableName: Table.Tasks.tableName,
         };
-        const result = (await this.dynamoDb.scan(params).promise()).Items?.map(e=>TaskSchema.parse(e) as Task)
+        const dbResponse = await this.dynamoDb.scan(params).promise()
+        const result = dbResponse.Items?.map(e=>TaskSchema.parse(e) as Task)
```

Comment:
you can inline that 
return dbResponse.Items?.map(e=>TaskSchema.parse(e) as Task) ?? []